crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -1206,6 +1207,87 @@ mod test {
         conductor_handle.shutdown().await.unwrap().unwrap();
     }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn dump_conductor_state() {
+        holochain_trace::test_run();
+        let uuid = Uuid::new_v4();
+        let dna = fake_dna_zomes(
+            &uuid.to_string(),
+            vec![("zomey".into(), TestWasm::Foo.into())],
+        );
+        let agent_pubkey = fake_agent_pubkey_1();
+
+        let _ = RealRibosomeFixturator::new(crate::fixt::curve::Zomes(vec![TestWasm::Foo]))
+            .next()
+            .unwrap();
+        let (_tmpdir, _, conductor_handle, _agent_key) = setup_app_in_new_conductor(
+            "test app".to_string(),
+            Some(agent_pubkey.clone()),
+            vec![(dna.clone(), None)],
+        )
+        .await;
+        let dna_hash = dna.dna_hash();
+
+        conductor_handle
```

Comment:
After adding an app interface, this test fails (without the included changes to conductor types & serializers).